frontend/src/pages/vpn-relay-welcome.page.tsx

```diff
@@ -27,22 +28,18 @@ const VpnRelayWelcome: NextPage = () => {
     authenticatedFetch("/accounts/profile/refresh");
   }, []);
 
-  // redirect user if they haven't purchased vpn-relay
-  if (profile && profile.data && !profile.data[0].has_vpn) {
-    router.push("/");
-  }
-
+  console.log(runtimeData.data);
```

Comment:
```suggestion
```

Reply:
Good catch! Perhaps we want to add this to our linter to catch?

Reply:
That is an excellent idea. Do you want to do it or shall I? (https://eslint.org/docs/latest/rules/no-console)